templates/bootstrap3-horizontal/components/quickForm/quickForm.js

```diff
@@ -5,10 +5,38 @@ Template['quickForm_bootstrap3-horizontal'].helpers({
   labelClass: function () {
     return this.atts["label-class"];
   },
+  fieldGroupLabel: function () {
+    var name = this.name;
+
+    // if field group name is of the form XY_abcde where "XY" is a number, remove prefix
+    if (typeof parseInt(name.substr(0,2)) === "number" && name.charAt(2) === "_") {
```

Comment:
You should always specify `10` for the second arg of `parseInt` because some browsers use base 2 or some other base.

Also, this check doesn't really work because `typeof parseInt('foo')` is "number", which is because `typeof NaN` is, paradoxically, "number". So instead it should be `if (!isNaN(parseInt(name.substr(0,2), 10)) && name.charAt(2) === "_")`.